experimental/framework/frs-client/src/test/FrsClient.spec.ts

```diff
@@ -3,4 +3,58 @@
  * Licensed under the MIT License.
  */
 
-// TODO: Add tests here that mock a connection to FRS
+import { strict as assert } from "assert";
+import { v4 as uuid } from "uuid";
+import { SharedMap, ContainerSchema } from "@fluid-experimental/fluid-framework";
+import { generateUser } from "@fluidframework/server-services-client";
+import {
+    FrsClient,
+    FrsConnectionConfig,
+    FrsContainerConfig,
+    InsecureTokenProvider,
+} from "..";
+
+describe("FrsClient", () => {
+    // use FrsClient will run against live service. Default to running Tinylicious for PR validation
+    // and local testing so it's not hindered by service availability
+    const useFrs = process.env.FLUID_CLIENT === "frs";
```

Comment:
i think this config stuff and useFrs checking would be better off in its own file (and exposing a createClient call or something), then the test can call that as part of setup

Reply:
Right, I think cleaning it up would make it look less messy. I went with creating a `CreateClient` class and calling the `create()` method will return a new `FrsClient` instance.